extension/js/common/browser/ui.ts

```diff
@@ -260,6 +260,21 @@ export class Ui {
         }
       });
     },
+    attachment: async (iframeUrl: string): Promise<void> => {
+      await Ui.swal().fire({
+        onOpen: () => {
+          $(Swal.getContent()).attr('data-test', 'attachment-dialog');
+          $(Swal.getCloseButton()).attr('data-test', 'dialog-close');
+        },
+        html: `<iframe src="${Xss.escape(iframeUrl)}" style="border: 0"></iframe>`,
```

Comment:
I recommend adding the `sandbox` attribute to this iframe. It adds a bunch of restrictions to an iframe ([here is a list of them](https://www.w3schools.com/tags/att_iframe_sandbox.asp)), such as treating the iframe's origin as unique, blocking script execution, and preventing form submissions.

Of course, none of those things should be possible now, but sandbox would make them extra not possible. I tried it locally and it doesn't break (appear to) break anything.

Reply:
Done, thanks :+1: